hypershift-operator/controllers/nodepool/aws.go

```diff
@@ -24,7 +24,6 @@ func awsMachineTemplateSpec(infraName, ami string, hostedCluster *hyperv1.Hosted
 	subnet := &capiaws.AWSResourceReference{}
 	if nodePool.Spec.Platform.AWS.Subnet != nil {
 		subnet.ID = nodePool.Spec.Platform.AWS.Subnet.ID
-		subnet.ARN = nodePool.Spec.Platform.AWS.Subnet.ARN
```

Comment:
we'll need to align here as well https://github.com/openshift/hypershift/blob/main/api/v1beta1/nodepool_types.go#L628
That can be a follow up.

Reply:
Thanks for catching that! I went ahead and removed it and added the changes to the `Remove ARN from AWS machine template generation` [commit](https://github.com/openshift/hypershift/pull/2000/commits/cec09b724709a022177faf7de743dc5390373a91).